src/tests/test_public.py

```diff
@@ -5,6 +5,9 @@
 from parameterized import parameterized
 from common import public_dir
 
+re_email = re.compile(r'^([a-zA-Z0-9_\-\.]+)@([a-zA-Z0-9_\-\.]+)\.([a-zA-Z]{2,5})$')
```

Comment:
Should we just have one `re_email` and put that into common and share it?

Reply:
Moved `re_email` to `common.py`.